spec/CLI.spec.js

```diff
@@ -220,7 +220,7 @@ describe('execution', () => {
 
   it('should start Parse Server', done => {
     const env = { ...process.env };
-    env.NODE_OPTIONS = '--dns-result-order=ipv4first';
+    // env.NODE_OPTIONS = '--dns-result-order=ipv4first';
```

Comment:
Isn't this handled by
https://github.com/parse-community/parse-server/blob/alpha/spec/helper.js#L10

Reply:
Yes, that may not be needed anymore, I disabled it to try without

Reply:
The tests fail when removing it, I think that is because the tests are spawning up a new process, hence this env var needs to be applied before that.